tachyon/math/finite_fields/packed_prime_field_base.h

```diff
@@ -134,12 +134,12 @@ class PackedPrimeFieldBase : public Field<Derived> {
   // MultiplicativeGroup methods
   std::optional<Derived> Inverse() const {
     Derived ret;
-    CHECK(PrimeField::BatchInverse(values_, &ret.values_));
+    CHECK(PrimeField::BatchInverseSerial(values_, &ret.values_));
```

Comment:
Is this because there's not much inverse to do?

Reply:
Exactly the operation inverses at most 16 elements at a time.